WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -224,10 +228,12 @@ public function is_targetted_token( $stackPtr ) {
 			}
 
 			if ( preg_match( $group['regex'], $token_content ) === 1 ) {
-				$this->process_matched_token( $stackPtr, $groupName, $token_content );
+				$skip_to = $this->process_matched_token( $stackPtr, $groupName, $token_content );
```

Comment:
Shouldn't this check the previous value of `$skip_to`, and only overwrite it if the new value is less? Assuming that different matches can return different values there. In practical terms, that would mean that the minimum number of tokens requested by the groups would be skipped, rather than just the last thing that happened to be returned there. And if any of the matches return null, then probably nothing should be skipped.

Though on further thought, perhaps different sniffs would want different behaviors here.